### ISSUE TITLE
Accuracy collapsed while setting dynamic  range to quantize Bert model

### ISSUE THREAD

**Issue by BillAmihom** [NVIDIA/TensorRT#3052]
## Description

I quantized the torch.Linear layer of the Bert model, and then converted the model from the torch model to the ONNX model ，and then parsed it into the tensorRT engine. The accuracy of my model collapsed. My method is to use TensorRT API layer.set_output_type() and tensor.dynamic_range on the network of the engine to set the  INT8 of layers and the dynamic_range of outputs.
(1) I am worried that “dynamic_range” will become invalid by the graph/layer fusion process of TensorRT. Will this problem happen?
(2) In addition, my torch.Linear layer may be converted into GEMM or MAT MUL+ADD the two operators in ONNX. For operators such as MAT MUL+ADD, my dynamic range is only set to the MatMul layer. (Because TensorRT will automatically combine MatMul and Add into GEMM in TensorRT), but in fact its dynamic range information is the original entire torch.Linear layer. Will this dynamic range be regarded as MatMul's dynamic range only?

## Environment

**TensorRT Version**: 

**NVIDIA GPU**: A100

**NVIDIA Driver Version**: 8.6

**CUDA Version**:11.7

**CUDNN Version**:

Operating System:

Python Version (if applicable):3.8

Tensorflow Version (if applicable):

PyTorch Version (if applicable):

Baremetal or Container (if so, version):

## Relevant Files

**Model link**:

## Steps To Reproduce

**Commands or scripts**:

**Have you tried [the latest release](https://developer.nvidia.com/tensorrt)?**:

**Can this model run on other frameworks?** For example run ONNX model with ONNXRuntime (`polygraphy run <model.onnx> --onnxrt`):

**Comment by ttyio** [comment]
closing since no activity for more than 3 weeks, pls reopen if you still have question, thanks!
